src/novel_applications/payment/1-to-n_private_payment.md

```diff
@@ -0,0 +1,68 @@
+# 1-to-n Private Payment
+
+![1-to-n private payment](../../assets/images/1-to-n_private_payment.png)
+
+## What is 1-to-n Payment and Why is it Powerful?
+
+1-to-n private payment is a powerful primitive enabling:
+
+- Consolidation of multiple payments in a single transaction
+- Increased privacy for all involved parties
+- Reduced processing time
+- Reduced complexity of tasks and possibility of errors
+- Reduced administrative costs
+
+The use cases for 1-to-n payments are diverse, encompassing a wide range of financial operations. These include user payments, large-scale corporation payrolls, and document, ticket, and NFT issuance, among others.
+
+1-to-n payment would enable actors from individual users to large corporations to benefit from a better payment primitive.
+
+### 1-to-n Payments for Companies
```

Comment:
Debating on wether to remove or merge with the first chapter the two "for companies" section here. 

Left them to give you an idea and get your advice.